sync/class.jetpack-sync-module-full-sync.php

```diff
@@ -242,17 +244,27 @@ function update_sent_progress_action( $actions ) {
 		foreach ( Jetpack_Sync_Modules::get_modules() as $module ) {
 			$module_actions     = $module->get_full_sync_actions();
 			$status_option_name = "{$module->name()}_sent";
+			$total_option_name  = "{$status_option_name}_total";
 			$items_sent         = $this->get_status_option( $status_option_name, 0 );
+			$items_sent_total   = $this->get_status_option( $total_option_name, 0 );
 
 			foreach ( $module_actions as $module_action ) {
 				if ( isset( $actions_with_counts[ $module_action ] ) ) {
 					$items_sent += $actions_with_counts[ $module_action ];
 				}
+
+				if ( ! empty( $actions_with_total_counts[ $module_action ] ) ) {
+					$items_sent_total += $actions_with_total_counts[ $module_action ];
+				}
 			}
 
 			if ( $items_sent > 0 ) {
 				$this->update_status_option( $status_option_name, $items_sent );
 			}
+
+			if ( $items_sent_total !== 0 ) {
```

Comment:
If so, then perhaps using the opportunity and fixing the statement above to be a Yoda condition as well is a good idea?

Reply:
That one does need to be ( https://github.com/Automattic/jetpack/pull/12125#discussion_r277715870 ), but the one above it doesn't since there's no way to accidentally assign `$items_sent_total` a value.

Reply:
I never got used to using Yoda conditions, and TBH I'm not a huge fan of them 😅 

Updated though, thanks. 😉

Reply:
Not a big fan myself, just thought this would be useful. :)